### PR TITLE
catch crash error in plugin

### PR DESCRIPTION
**Pull request by houqp**
for https://github.com/koreader/koreader/issues/2790

### CODE REVIEW

File: frontend/apps/filemanager/filemanagermenu.lua [hunk 1]
```diff
@@ -8,7 +8,9 @@ local InputDialog = require("ui/widget/inputdialog")
 local Screensaver = require("ui/screensaver")
 local Search = require("apps/filemanager/filemanagersearch")
 local SetDefaults = require("apps/filemanager/filemanagersetdefaults")
+local logger = require("logger")
```
Comment (Frenzie) [inline frontend/apps/filemanager/filemanagermenu.lua]: nitpick (should go under dbg)

**Comment by Frenzie** [comment]
Hm, is there a direct way to test this in busted or should I set up a second Travis environment when I have the time? I just mean to add a second matrix line [here](https://github.com/koreader/koreader/blob/d6c81f5097702f246f35e7b42a9569b56e75e7c4/.travis.yml#L14) that'd result in some slightly different script paths, nothing fancy.

**Comment by houqp** [comment]
Merging this since it's a critical fix

**Comment by houqp** [comment]
@Frenzie yes you should be able to test this in busted by manually introduce a crash in a mock plugin and try running it through those protected code paths like registerToMainMenu. You don't need a separate env or .ci folder. You can just write another busted test. One thing that needs to be done in that test though is to turn off debug. Because guard would be enabled in debug mode and enforce the crash to happen immediately.

**Pull request merged**